src/d3d12/d3d12-device.cpp

```
@@ -347,15 +494,30 @@ Result DeviceImpl::initialize(const DeviceDesc& desc, BackendImpl* backend)
+            HRESULT result = m_D3D12CreateDevice(m_dxgiAdapter, featureLevel, IID_PPV_ARGS(m_device.writeRef()));
+            createDeviceAttempts[createDeviceAttemptCount++] = {featureLevel, result};
+            if (SUCCEEDED(result))
             {
                 break;
             }
         }
         if (!m_device)
         {
+            auto adapters = backend->getAdapters();
+            int64_t adapterIndex = -1;
+            if (adapter && adapters.data())
```

Cppcheck (2.20.0)

[style] The function 'writeCommand' is never used.

(unusedFunction)